libs/wingsdk/src/std/json.ts

```diff
@@ -32,6 +32,32 @@ export class Json {
     throw new Error("Macro");
   }
 
+  /**
+   * Returns the values from the Json.
+   *
+   * @macro (Object.values($args$))
+   *
+   * @param json to get values from
+   * @returns the values from the Json as array of Json
+   */
+  public static values(json: Json): Json[] {
+    json;
+    throw new Error("Macro");
+  }
+
+  /**
+   * Returns the entries from the Json.
```

Comment:
Might be worthwhile to add an example to make it clear that it's an array of arrays.

Reply:
I actually added an example in the Library Reference: https://github.com/winglang/wing/pull/4092/files#diff-930e1858fb1950a337183b7f181fd0706fac2a9daf0e5b135b884e09cbaf6fe7R242-R251